apps/src/AzureTextToSpeech.js

```diff
@@ -314,12 +331,18 @@ export default class AzureTextToSpeech {
    * @private
    */
   createSoundResponse_ = opts => {
+    const onEnded = () => {
+      if (opts.onComplete) {
+        opts.onComplete();
+      }
```

Comment:
nit: we can use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) to only call this function if it's present -- `opts.onComplete?.();`